stages/org.osbuild.anaconda

```diff
@@ -33,24 +56,31 @@ CONFIG = """
 # osbuild customizations
 
 [Anaconda]
-# List of enabled Anaconda DBus modules
-kickstart_modules =
 """
 
 
 def main(tree, options):
-    modules = options["kickstart-modules"]
     product_dir = os.path.join(tree, "etc/anaconda/conf.d")
     os.makedirs(product_dir, exist_ok=True)
 
-    with open(os.path.join(product_dir, "90-osbuild.conf"), "w", encoding="utf8") as f:
+    with open(
+        os.path.join(product_dir, "90-osbuild.conf"), "w", encoding="utf8"
+    ) as f:
         f.write(CONFIG)
-        for m in modules:
-            f.write(f"    {m}\n")
+
+        for grouping in [
+            "kickstart-modules",
+            "activatable-modules",
+            "forbidden-modules",
+            "optional-modules",
+        ]:
+            if grouping in options:
```

Comment:
I'm not following the logic here entirely.  Couldn't we just iterate over `options` instead?  Or is this done to preserve the order?

Reply:
Mostly to whitelist the accepted sections.